machine/dni/dni_3048up/INSTALL

```diff
@@ -0,0 +1,92 @@
+============================================================
+Installing ONIE on DNI 3048UP
+============================================================
+
+Cross-Compiling ONIE
+====================
+
+Change directories to ``build-config`` to compile ONIE.
+
+To compile ONIE first change directories to ``build-config`` and then
+type ``"make MACHINEROOT=../machine/dni/ MACHINE=dni_3048up all"``.
+For example::
+
+  $ cd build-config
+  $ make -j4 MACHINEROOT=../machine/dni/ MACHINE=dni_3048up all
+
+When complete, the ONIE binaries are located in
+``build/images``::
+
+  -rw-r--r-- 3604480 May  7 11:38 onie-dni_3048up-r1.bin
+
+Cross-Compiling the DEMO OS
+===========================
+
+To compile the demo OS first change directories to ``build-config``
+and then type ``"make MACHINEROOT=../machine/dni/ MACHINE=dni_3048up demo"``.
+
+  $ cd build-config
+  $ make -j4 MACHINEROOT=../machine/dni/ MACHINE=dni_3048up demo
+
+When complete, the demo OS installer is located in ``build/images``::
+
+  -rw-r--r-- 10486945 Jun 24 13:30 demo-installer-arm-dni_3048up-r1.bin
+
+Installing the ONIE binaries
+============================
+
+Step 1 -- Put the ONIE files on a TFTP server
+---------------------------------------------
+
+The following directions assume the files are on the root of the TFTP
+server.
+
+Step 2 -- Install ONIE kernel (dni_3048up-r1.u-boot dni_3048up-r1.itb)
+-----------------------------------------------
+
+Copying the image down using TFTP and flash to the SPI flash::
+
+  =>tftp $loadaddr dni_3048up-r1.u-boot
+  =>sf probe 0;
+  =>sf erase 0 0x100000;
+  =>sf write $loadaddr 0 $filesize;
+  
+  =>tftp $loadaddr dni_3048up-r1.itb
```

Comment:
Please use ONIE generated binaries after fixing onie-rom.conf instead of itb.
Refer to comments at line 20